content/02.introduction.md

```diff
@@ -2,19 +2,19 @@
 
 <!-- ### General introduction to the use of collaborative computational resources by scientists -->
 <!-- *Contributors to this section: PHPB* -->
-Most scientists, including ecologists and evolutionary biologists, are increasingly dependent on computational tools in their research [see @doi:10.1109/SECSE.2009.5069155].
-Researchers now write and use code as part of their scientific workflow to perform a wide-variety of tasks ranging from data management, data analysis, study replication, to the application and the development of tools for hypothesis testing.
+Most scientists, including ecologists and evolutionary biologists, are increasingly dependent on computational tools in their research [@doi:10.1109/SECSE.2009.5069155].
```

Comment:
Should this be EEB here?

Reply:
I think it's OK to leave as is because it's very first mention of EEB in the paper (other than the abstract).  That being said in the last sentence of this paragraph we say "ecology and evolutionary **biology** (EEB). Not sure whether we should move the abbreviation to first sentence or leave in last sentence of this paragraph since first mention refers to biologists and later refers to biology.